src/app/modules/form/admin-form/__tests__/admin-form.controller.spec.ts

```diff
@@ -3900,7 +3900,7 @@ describe('admin-form.controller', () => {
         MOCK_FORM,
         MOCK_USER_ID,
         expectedParams,
-        { duplicateStripped: true },
+        { overrideEmails: ['andanother@example.com'] },
```

Comment:
nice covering that if use template is used - it will provide the `overrideEmails` from the user (form owner)! 👍 

this gives us confidence that as long as the CI passes, TC2 should be covered